### ISSUE TITLE
Log In failure

### ISSUE THREAD

**Issue by RichDG** [olsh/Feedly-Notifier#168]
Hi. Sorry but my question got lost in a previous thread. I am sure the answer is just no help is possible to get older versions or authentication tokens so apologies for any time you may feel I am wasting of yours but just wanted a definitive answer as the other threads "solution" was to install the latest version which isn't possible in my case due to the old version of firefox being used.

To recap - I have to use an old version of Firefox (47.0.1) and consequently an old version of Feedly Notifier (2.11.2). All worked fine until the last day or so and now I cannot log in any more. 

I click on log in and it takes me to the page for the 3 different ways to log in and whether using a feedly account, google or firefox I still can't get an authentication token to re-start my feedly notifier.

Is there a way to get this old version (2.11.2) and try re-installing it (web search says no version this old available anywhere I can see sadly) or another way to get the authentication to log in to my account and retrieve my feeds ? 

Thanks for any help and a great app (even though I have a feeling the answer will be no and I can no longer use it). My pc is windows 10 home 64-bit and fully updated bar the last March update that cannot be installed yet due to conflicts.

**Comment by olsh** [comment]
Hi @RichDG ,

Here is the old version.
[feedly_notifier-2.11.2-fx.zip](https://github.com/olsh/Feedly-Notifier/files/4956088/feedly_notifier-2.11.2-fx.zip)

In case if the reinstall doesn't solve the problem, I suggest removing Feedly Notifier settings from FF. I believe the settings can be found on `about:config` page, but I'm not sure. As a last resort, you can try to create a new FF profile.

**Comment by RichDG** [comment]
You have nothing to apologize for at all, most wouldn't even bother to reply to a question about a version so far out of date so I appreciate you replying and trying to help.

Unfortunately re-installing the old version didn't work, even after removing all trace in about:config as well as the basic uninstall itself. 

I had tried a new profile, creating one directly with feedly but to no avail.

Unfortunately for whatever reason logging back in seems to not be possible. I should consider myself lucky it worked for so long on such an old version of firefox.

It looks like there is no solution. Thanks so much for taking the effort to reply, it is greatly appreciated. It's a great add on which I look forward to using when I am no longer using FF47 at some point in the future.

Thanks again.

**Comment by olsh** [comment]
I just installed FF47 and surprisingly the old version of the extension works.
I believe that the problem is in invalid token. So, we need to remove the old token somehow.

Have you tried to go to `about:addons` > `Feedly Notifier settings` and click the `Logout` button at the bottom? The `Logout` button should delete the old token.
Also, try to create a clean FF profile as described here https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles#w_creating-a-profile and install the extension.

**Comment by RichDG** [comment]
Thank you for persevering ! I logged out at the bottom and re-started the browser and before even trying the new profile bit I clicked the log in bit, it went to the same page for installing the add on and looked like nothing had happened...and then suddenly the notifier wasn't greyed out any more and had news items.

Thank you so much for all the help, I am not sure exactly which combo of re-installing the add on, logging out at the bottom, clearing all details from about:config etc worked the magic to get it working but I do know without your help it wouldn't be working now.

Thanks again for all your help. :-)
